spec/app/ics-interchain-queries/README.md

```diff
@@ -0,0 +1,146 @@
+---
+ics: TBA
+title: Interchain Query
+stage: draft
+category: IBC/APP
```

Comment:
should this be under APP? Considering we're not implementing ICS26 here

Reply:
To me I feel like it falls best under app (it's essentially an extension of interchain accounts) but if you think otherwise then?